build/nse/vl3-nse/README.md

```diff
@@ -29,12 +29,12 @@ This is an NSE that creates a L3 routing domain between NSC workloads. Each NSE
 1. This currently only works with a custom version of the NSM installation.
 
     ```sh
-    mkdir -p $GOPATH/src/github.com/cisco-app-networking
-    git clone https://github.com/cisco-app-networking/networkservicemesh
+    mkdir -p $GOPATH/src/cisco-app-networking.github.io/nsm-nse
```

Comment:
don't update paths in this file and related ones

Reply:
check